pandas/core/arrays/_ranges.py

```diff
@@ -32,55 +35,40 @@ def generate_regular_range(
 
     Returns
     -------
-    ndarray[np.int64] representing nanosecond unix timestamps
+    ndarray[np.int64]
+        Representing nanosecond unix timestamps.
     """
-    if isinstance(freq, Tick):
```

Comment:
so are we now only getting here with a `Tick`?  if so, can you update the docstring and annotation

Reply:
Yes, we are now only getting here if freq is an instance of `Tick` now.